cognite_toolkit/cdf_tk/load.py

```diff
@@ -296,43 +316,33 @@ def get_required_capability(cls, ToolGlobals: CDFToolConfig) -> Capability | lis
 
     @classmethod
     @abstractmethod
-    def get_id(cls, item: T_CogniteResource | T_CogniteResourceWrite) -> T_ID:
+    def get_id(cls, item: T_WriteClass | T_WritableCogniteResource) -> T_ID:
         raise NotImplementedError
 
-    @staticmethod
-    def fixup_resource(local: T_CogniteResourceWrite, remote: T_CogniteResource) -> T_CogniteResourceWrite:
-        """Takes the local (to be pushed) and remote (from CDF) resource and returns the
-        local resource with properties from the remote resource copied over to make
-        them equal if we should consider them equal (and skip writing to CDF)."""
-        return local
```

Comment:
This is no longer necessary as it implemented through the `cognite-sdk` with the `.as_write` method